pkg/kubelet/cm/cpumanager/cpu_manager_test.go

```diff
@@ -61,31 +61,30 @@ func (s *mockState) GetCPUSetOrDefault(podUID string, containerName string) cpus
 	return s.GetDefaultCPUSet()
 }
 
-func (s *mockState) SetCPUSet(podUID string, containerName string, cset cpuset.CPUSet) {
+func (s *mockState) SetCPUSet(podUID string, containerName string, cset cpuset.CPUSet, defaultCPUSet cpuset.CPUSet) {
 	if _, exists := s.assignments[podUID]; !exists {
 		s.assignments[podUID] = make(map[string]cpuset.CPUSet)
 	}
 	s.assignments[podUID][containerName] = cset
+	s.defaultCPUSet = defaultCPUSet
```

Comment:
can we compute the updated `defaultCPUSet` automatically in these functions which manipulate the assignments instead of passing from the outside? this will reduce the API changes and will reduce the coupling with the calling code (applies also below)